### ISSUE TITLE
Add a hold Button

### ISSUE THREAD

**Issue by ju-w** [maks/droid-synth#9]
Hello!

This App is absolutely amazing, but it _desperately_ needs a **hold Button** (Toggle)!

Eg: Hold active: Tap key -> Sound is continuing to play after release; Tap key again -> Sound will stop.
Hold inactive: Same behavior as currently

This would make the App 1000x better!

**Comment by maks** [comment]
Thank you for raising this issue. 
I know about Midi after touch, but I'm not familiar with any synths that have a note hold feature - would be able to point me to any examples you know of?

**Comment by ju-w** [comment]
I only installed DRC from the playstore but I think most apps have it.

**Comment by bunthut** [comment]
Hasn't a sustain pedal a quite similar function? Synthesizers with sustain pedals do exist at least: https://www.amazon.com/Korg-Nautilus-Performance-Workstation-Polishing/dp/B09TRSW6N1
 for example.

But actually I was coming here because I love the app and if there's anything I could help :) I'm not really into app development, but python and I do art stuff, if any media or design is needed.

**Comment by maks** [comment]
Yes a sustain pedal would be a bit similar, a sustain pedal might even already work via MIDI controller, I just don't have one to try out, but I don't think that's what is being requested here.
